cdap-ui/app/tracker/controllers/audit-ctrl.js

```diff
@@ -118,9 +119,120 @@ class TrackerAuditController {
         });
       });
   }
+
+  showMetadataChangeInfo(logData) {
+    this.$uibModal.open({
+      templateUrl: '/assets/features/tracker/templates/partial/metadata-change-info.html',
+      size: 'lg',
+      backdrop: true,
+      keyboard: true,
+      windowTopClass: 'tracker-modal metadata-info-modal',
+      controller: metadataChangeInformation,
+      controllerAs: 'MetadataInfo',
+      resolve: {
+        logDataObj: () => {
+          return logData;
+        }
+      }
+    });
+  }
+}
+
+function metadataChangeInformation(logDataObj, avsc, SchemaHelper) {
+  'ngInject';
+
+  let allFormattedData = {},
+    tags= {},
+    schema= {},
+    properties = {};
+
+  let getSchemaArray = (schemaToParse) => {
+    if(!schemaToParse) {
+      return [];
+    }
+
+    let schemArr = [];
+    let parsed = avsc.parse(schemaToParse, { wrapUnions: true });
```

Comment:
@elfenheart can this throw an exception? Should we wrap this in a try catch block?